docs/docusaurus/docs/guides/connecting_to_your_data/fluent/batch_requests/how_to_request_data_from_a_data_asset.md

```diff
@@ -59,18 +53,14 @@ You can create a dictionary of keys pulled from the `batch_request_options` tupl
 
 We will use the `build_batch_request(...)` method of our Data Asset to generate a Batch Request.
 
-```python title="Python code"
-my_batch_request = my_asset.build_batch_request()
+```python name="tests/integration/docusaurus/connecting_to_your_data/fluent_datasources/get_existing_data_asset_from_existing_datasource_pandas_filesystem_example.py my_batch_request"
 ```
 
 ### 5. Verify that the correct Batches were returned
 
 The `get_batch_list_from_batch_request(...)` method will return a list of the Batches a given Batch Request returns.  Because Batch definitions are quite verbose, it is easiest to determine what data the Batch Request will return by printing just the `batch_spec` of each Batch.
 
-```python title="Python code"
-batches = datasource.get_batch_list_from_batch_request(my_batch_request)
```

Comment:
This was the mistake (perhaps a typographical error) that launched the revision, leading to the proposed changes.